source/world_builder/features/subducting_plate_models/temperature/mass_conserving.cc

```diff
@@ -169,6 +170,13 @@ namespace WorldBuilder
           prm.declare_entry("reference model name",  Types::String("half space model"),
                             "The type of thermal model to use in the mass conserving model of slab temperature. "
                             "Options are half space model and plate model");
+
+          prm.declare_entry("apply spline",  Types::Bool(false),
+                            "Whether a spline should be applied on the mass conserving model.");
```

Comment:
change  "Whether a spline should be applied on the mass conserving model."
to "Choose to apply a spline to smooth temperature where the two analytical equations meet at the minimum temperature."